src/main/java/com/ironcorelabs/tenantsecurity/kms/v1/TenantSecurityErrorCodes.java

```diff
@@ -4,75 +4,90 @@
 import java.util.Map;
 
 /**
- * Enum of error codes that can occur as part of requests to the Tenant Security Proxy Docker container.
+ * Enum of error codes that can occur as part of requests to the Tenant Security Proxy Docker
+ * container.
  */
 public enum TenantSecurityErrorCodes {
 
-    // map to TspServiceException
-    UNABLE_TO_MAKE_REQUEST(0, "Request to Tenant Security Proxy could not be made"),
-    UNKNOWN_ERROR(100, "Unknown request error occurred"),
-    UNAUTHORIZED_REQUEST(101, "Request authorization header API key was incorrect."),
-    INVALID_REQUEST_BODY(102, "Request body was invalid."),
+  // map to TspServiceException
+  UNABLE_TO_MAKE_REQUEST(0, "Request to Tenant Security Proxy could not be made"), UNKNOWN_ERROR(
```

Comment:
This block will need manual formatting unless we can do something to get to something reasonable.